### PR TITLE
Add first pass at content

### PR DESCRIPTION
**Pull request by kylefiedler**

### CODE REVIEW

File: source/index.html.haml [hunk 1]
```diff
@@ -9,6 +9,49 @@
...
+    %p After the conference you'll have plenty to see and do as the venue is located in the middle of world famous Sixth Street. Made up of dozens of bars and venues, you're guaranteed to hear a wide variety of live music as you walk down the street which is closed for pedestrian traffic for weekends. Known as "The music capital of the world" Austin is where musicians come to be discovered and discover one another.
+
+%section#speakers
+  %header
+    %h2 Speakers
+    %a{ href: "#" } Call for Proposals
+
+  %ul
+    %li
+      %img{ src: "https://pbs.twimg.com/profile_images/635576395565953024/5bhOBmOu.jpg", alt: "" }/
```
Comment (ethangl) [inline source/index.html.haml]: You can use `image_tag`

**Comment by ethangl** [comment]
🐑

File: source/index.html.haml [hunk 2]
```diff
@@ -9,6 +9,49 @@
   %span RUBY
   %span WEIRD
 
+%header
+  %address
+    %span Alamo Drafthouse Ritz
+    %span 320 E 6th St.
+    %span Austin, TX 78701
+  .date
+    %span Friday, October 23, 2016
```
Comment (calebhearth) [inline source/index.html.haml]: 28th

File: source/index.html.haml [hunk 3]
```diff
@@ -9,6 +9,49 @@
...
+    %a{ href: "#" } Sponsor KRW 2016
+  %ul
+    %li
+      %h3 thoughtbot
+
+%section#why-austin
+  %h2 Why Austin?
+  %p Austin is our home and it's at the heart of how we live and write software. Known for it's eclectic attitude, the city maintains the unofficial motto: "Keep Austin Weird'. The people are friendly, accepting of artistic and individual expression, lending the city a vibrant and progressive energy. We believe that as developers we grow stronger through play and exploration of our craft. In Austin, this belief is strengthened and supported as we see companies care as much about personal growth as their bottom lines. We wanted to throw a conference that feels like Austin, while taking the opportunity to show off what it is that makes our city great.
+
+%section#sponsors
```
Comment (calebhearth) [inline source/index.html.haml]: Wrong header maybe

**Pull request merged**